### PR TITLE
Conform Gherkin defaults to original TMK keymap

### PR DESCRIPTION
**Pull request by sowbug**
Updates the default QMK keymap for Gherkin to [the same one](http://www.keyboard-layout-editor.com/#/gists/7eac308ec268b36b0621bfca7500d20c) that's in [the original TMK repository](https://git.40percent.club/di0ib/tmk_keyboard/src/branch/master/keyboard/gherkin/actionmap_gherkin.c). This will let new Gherkin builders flash the default QMK firmware and get the same experience that they would if they built and flashed from di0ib's TMK repository.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/40percentclub/gherkin/keymaps/default/keymap.c [hunk 1]
```diff
@@ -1,21 +1,54 @@
...
+  [3] = LAYOUT_ortho_3x10(
+    _______, _______, _______, _______, _______, KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
+    KC_TAB,  _______, _______, _______, _______, KC_COMM, KC_DOT,  KC_SLSH, KC_SCLN, KC_QUOT,
+    _______, _______, _______, _______, _______, _______, KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT
+  ),
+
+  [4] = LAYOUT_ortho_3x10(
+    _______, _______, _______, _______, _______, KC_UNDS, KC_PLUS, KC_LCBR, KC_RCBR, KC_PIPE,
+    KC_TAB,  _______, _______, _______, _______, KC_LABK, KC_RABK, KC_QUES, KC_COLN, KC_DQUO,
+    _______, _______, _______, _______, _______, _______, KC_HOME, KC_PGUP, KC_PGDN, KC_END
```
Comment (noroadsleft) [inline keyboards/40percentclub/gherkin/keymaps/default/keymap.c]: Page Up and Page Down should be the other way around, per your original TMK link.

```suggestion
    _______, _______, _______, _______, _______, _______, KC_HOME, KC_PGDN, KC_PGUP, KC_END
```

Comment (sowbug) [reply keyboards/40percentclub/gherkin/keymaps/default/keymap.c]: Yipe! Coding too late at night. Fixed in https://github.com/qmk/qmk_firmware/pull/8186/commits/2bdd9e53874bd3b062d4e7d274a65828a7c7e7fe. Thanks for the catch.

**Comment by sowbug** [comment]
There were two mistakes in the original PR, and now after a good night's sleep I've rechecked the source and the actual behavior on a real device, I've fixed them. I think this PR is now ready.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**